src/Slot-Tests/ComputedSlotTest.class.st

```diff
@@ -37,28 +35,26 @@ ComputedSlotTest >> testReadComputedSlotReflective [
 { #category : #tests }
 ComputedSlotTest >> testReadComputedSlotReflectiveNoArg [
 	| slot object |
-	aClass := self make: [ :builder | builder slots: {(#slot1 => ComputedSlot with: [ 3 + 2 ] )} ].
+	slot := #slot1 => ComputedSlot with: [:obj | 3 + 2 ].
```

Comment:
Shouldn't this test use no args in the block? otherwise it's the same as `testReadComputedSlotReflective`

Reply:
fixed